### ISSUE TITLE
NTR: Pupillary response low to hydroxyamphetamine (Cranial nerves)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7801]
New term request Pupillary response low to hydroxyamphetamine (elthat_130225200433): Pupils

**Comment by pnrobinson** [comment]
@MickeySegal
would this be for Horner's syndrome?
"The pupillary response to hydroxyamphetamine eye-drops has been advocated as a test for sub-dividing patients with Horner's syndrome into those with lesions of the 1st or 2nd sympathetic neurone (normal dilatation) and those with lesions of the 3rd neurone (no or minimal dilatation). "

**Comment by MickeySegal** [comment]
We got it from https://www.ncbi.nlm.nih.gov/books/NBK1474/ "Somewhat small pupils that respond to light and accommodation but not to hydroxyamphetamine. Parasympatholytics dilate the pupils appropriately."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000994
- label: Lack of pupillary response to hydroxyamphetamine
- parents: [Abnormal pupillary function (HP:0007686)](https://hpo.jax.org/app/browse/term/HP:0007686)
- pmid: PMID:20301647
- def: 4-hydroxyamphetamine is used in eye drops to dilate the pupil, e.g.., to induce mydriasis for fundus examinations. This term refers to a lack of response of the pupil to topical application of 4-hydroxyamphetamine.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7801
- synonyms:
